extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/QuarkusJaegerTracer.java

```diff
@@ -6,9 +6,11 @@
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;
+import io.opentracing.util.ThreadLocalScopeManager;
 
 public class QuarkusJaegerTracer implements Tracer {
 
+    static final String LOG_TRACE_CONTEXT = "JAEGER_LOG_TRACE_CONTEXT";
     private static volatile Tracer tracer;
```

Comment:
I have noticed that changes to Jaeger configuration are not applied in dev mode (`maven quarkus:dev`). This is because tracer is a singleton. This could be fixed in a separate PR.